eldritch/test_usables.py

```diff
@@ -374,6 +376,101 @@ def testCannotBeCastOutsideOtherworldMovement(self):
   # TODO: a test covering the ability to cast after travelling during the movement phase
 
 
+class FleshWardTest(EventTest):
+  def setUp(self):
```

Comment:
Two additional tests that I would recommend - one where you use a Food to reduce the stamina loss from 1 to 0, and then validate that flesh ward is not castable.
The other one would be a test where you fail to cast flesh ward, and validate that it is exhausted and no longer usable.

Reply:
Addendum: add a TODO for a test to discard this if the ancient one awakens.